storageClient/storageClient_test.go

```diff
@@ -19,9 +19,10 @@ import (
 )
 
 var (
-	cyclopsServiceHost = os.Getenv("TOZNY_CYCLOPS_SERVICE_HOST")
-	testCtx            = context.Background()
-	ClientServiceHost  = os.Getenv("CLIENT_SERVICE_HOST")
+	cyclopsServiceHost    = os.Getenv("TOZNY_CYCLOPS_SERVICE_HOST")
+	testCtx               = context.Background()
+	ClientServiceHost     = os.Getenv("CLIENT_SERVICE_HOST")
+	FakeEncryptedGroupKey = "SZDkQQvA6srSZBy9tdYMAtgEE86fUzgLBnxP5bpz3U3bKiTWFllHGzpVU4o_VNNe._-U2eZWO0sKIZw0djuUCfPnvQHo72guB"
```

Comment:
why

Reply:
my check requires the bytes to be 97, since thats how many bytes a group key is but the way i was currently testing adding a group the encrypted group pair doesn't have the same number of bytes so i needed a way to allow my tests to work without having to go back and create membership keys for everything

Reply:
Here's my take: solve the problem (tediousness of creating a membership key) now so future developers of this code (including yourself) completely and correctly now so its not someone else's problem in the future.
Create a e3db-clients-go client/helper/test method/function that can create a valid membership key, and then use that wherever in the test you need to create a valid key. Once you solve this problem for yourself, you will have then provided a good foundation for other developers both external and internal to be able to use our product without having to work around any rough edges. Self love = customer love.

Reply:
working on this now and will transfer to storage service!